python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -195,14 +272,14 @@ def test_binary_ufunc_series_array(ufunc, has_nulls, indexed, type_, reflect):
             if type_ == "cupy" and reflect:
                 assert (cp.asnumpy(g) == e).all()
             else:
-                assert_eq(g, e)
+                assert_eq(g, e, check_exact=False)
```

Comment:
Is `check_exact` being added because the outputs are being different now?

Reply:
Yes. To be honest I think I was just lucky that they weren't different before. There is no reason to expect exact equivalence between the two results on different hardware here, I just didn't run into it when I wrote the tests for the previous PRs.